components/SiteFooter.vue

```diff
@@ -1,10 +1,10 @@
 <template>
-  <v-footer fixed app dark color="accent">
-    <v-col fluid>
+  <v-footer fixed app dark color="footer">
+    <v-col fluid align-content="center" align="center">
       <v-row>
         <v-col>
           <v-icon>mdi-needle</v-icon>
-          {{ $t("footer.siteTitle", [new Date().getFullYear()]) }}
+          {{ "©" + new Date().getFullYear() }}
```

Comment:
We can't copyright this is we want Netlify to approve us as open source

Reply:
Ohh! Can we just remove that whole column then?